lib/src/storage/core_store_sem_impl.dart

```diff
@@ -14,8 +14,16 @@ class CoreStoreSembastImp implements CoreStore {
       if (!parseIsWeb &&
           (Platform.isIOS || Platform.isAndroid || Platform.isMacOS))
         dbDirectory = (await getApplicationDocumentsDirectory()).path;
+      assert(() {
+        if (parseIsWeb) {
+          print(
+              'Warning: CoreStoreSembastImp of the Parse_Server_SDK does not encrypt the database on WEB.');
```

Comment:
Awesome :D

Reply:
I found this on stackoverflow.
I have to remember this trick, it's super useful.